### DISCUSSION TITLE
bug: Unneeded space after logo if empty message

### DISCUSSION THREAD

**Discussion by goyalyashpal** [badges/shields#10628] [General] [answered]
hi!
thanks for such a wonderful service making such a nice use of https://simpleicons.org . 

### titles

* bug: Unneeded space after logo if empty message
* badge: get JUST the logo, without ANY message or extra space

### summary

* the suggestion is to not put any extra space when the message is empty
* this is useful in case only the logo in the inline shield style is needed

### example

notice the good enough amount of space to the right of the logo in both the styles, i want to get rid of that

* URL params specified:

  ```
  https://img.shields.io/badge/-A8B9CC?logo=c&style=for-the-badge&logoSize=auto&logoColor=FFF
  https://img.shields.io/badge/-A8B9CC?logo=c&logoSize=auto&logoColor=FFF
  ```

* The outcome: 

  [![c](https://img.shields.io/badge/-A8B9CC?logo=c&style=for-the-badge&logoSize=auto&logoColor=FFF) CLANG](http://example.org)
  [![c](https://img.shields.io/badge/-A8B9CC?logo=c&logoSize=auto&logoColor=FFF) CLANG](http://example.org)

**Comment by LitoMore** [comment, marked as answer]
I just opened PR for fixing this issue:

- https://github.com/badges/shields/pull/10794
